boot/freeldr/freeldr/arch/drivers/hwide.c

```diff
@@ -906,6 +907,7 @@ IdentifyDevice(
                         break;
                     }
                 }
+                ASSERT(SectorSize != 0);
```

Comment:
In principle, having SectorSize == 0 is fine, see the case in line 863, but if so then a flag should certainly be set... ?
@disean @binarymaster , any ideas about that?

Reply:
That flag is only meaningful for ATAPI devices.